### ISSUE TITLE
path parameter in the service is not created, only referenced

### ISSUE THREAD

**Issue by lgazo** [ferdikoomen/openapi-typescript-codegen#315]
**Describe the bug**

Following method is generated as part of the `Service.ts` for an endpoint defined like this: `/manage/applications/{applicationId}/addons/{addonSystemKey}`. There are two path parameters. The snippet references variables but such variables are not present in the method signature.

```
public static async installAddonCommand(
        requestBody: InstallAddonCommand,
    ): Promise<AddonInstanceValue> {

        const result = await __request({
            method: 'post',
            path: `/manage/applications/${applicationId}/addons/${addonKey}`,
            body: requestBody,
        });
```

**Comment by ferdikoomen** [comment]
@lgazo Can you send me the spec file for this service? With this info it's hard to figure out whats going on

**Comment by lgazo** [comment]
sry, here it is... I cannot share whole definition, this is the part 

```
'/manage/applications/{applicationId}/addons/{addonSystemKey}':
	post:
	    requestBody:
		content:
		    application/json:
		        schema:
		            $ref: '#/components/schemas/InstallAddonCommand'
		required: true
	    responses:
		'201':
		    content:
		        application/json:
		            schema:
		                $ref: '#/components/schemas/AddonInstanceValue'
		            examples:
		                standard response:
		                    value:
		                        id: abc
		    description: Good good
		'404':
		    description: Bad bad
	    operationId: InstallAddonCommand
	parameters:
	    -
		name: applicationId
		schema:
		    $ref: '#/components/schemas/SystemIdValue'
		in: path
		required: true
	    -
		name: addonSystemKey
		schema:
		    $ref: '#/components/schemas/AddonSystemKeyValue'
		in: path
		required: true
```

**Comment by ferdikoomen** [comment]
@lgazo I just uploaded a new version with the fix (0.4.10). Parameters that where part of the path (not the operation) where indeed not added to the service, this is fixed now. You can update to the latest version, or make the parameters part of the operation for now. Cheers, Ferdi

**Comment by lgazo** [comment]
I have tried it and it generates it correctly now. Thank you a lot.
